azure-pipelines/build/templates/dosdkcpp-native-steps.yml

```diff
@@ -49,12 +64,10 @@ steps:
     workingDirectory: '/tmp/build-deliveryoptimization-sdk/linux-${{parameters.config}}'
   displayName: 'Install libdeliveryoptimization Debian Package'
 
-- task: PythonScript@0
-  condition: eq('${{parameters.skipTests}}', false)
```

Comment:
Why remove this condition? No need to build tests in release mode.

Reply:
oops, good catch